pom.xml

```diff
@@ -167,10 +176,26 @@
                     <version>${maven-dependency-plugin.version}</version>
                 </plugin>
                 <plugin>
-                    <groupId>dev.ikm.maven</groupId>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-clean-plugin</artifactId>
```

Comment:
All of these standard Maven plugin dependencies seem like something that should be inherited from the BOM - maybe this can be refactored later.